problem3/problem3_spec.rb

```diff
@@ -0,0 +1,8 @@
+require "./problem3"
+describe Problem3 do
+	it "Dublicate Numbers" do
+		example3 = Problem3.new(1,3,1,4,3,4,8,5)
+		example3.DublicateNumbers;
+		expect(example3.array).to  eql([1,3,4]);
```

Comment:
Unfortunately this is not correct, the expected result is a boolean, read the requirement again and update your solution.